lib/CmdMessenger/CmdMessenger.h

```diff
@@ -232,6 +232,17 @@ class CmdMessenger
       }
   }
   
+	/**
+	 * Send a single argument as string w/o field_separator
+	 *  Note that this will only succeed if a sendCmdStart has been issued first
+	 */
+	template < class T > void sendSingleArg(T arg)
```

Comment:
Is there a better name for this? Maybe "sendRawByte"?

Reply:
Maybe just "sendArg()"?
All other functions habe "Cmd" and "Arg" in its name. This function shall send w/o a command, so just let it away.

Reply:
How about `sendPreformattedArg()`? Then it follows the same naming pattern and is also really clear what it does.

Reply:
If a have a look on all send...() functions, they seem all to be preformatted. I think the comparible (existing) function is sendCmdArg(), but here w/o the command. So just as my previous proposal sendArg().
Maybe directly to add:
`void sendArg(T arg, unsigned int n);
void sendSciArg(double arg, unsigned int n = 6);
void sendBinArg(T arg);
`
like the other ones in the lib.

Reply:
Works for me!